plugins/TwoFactorAuth/lang/en.json

```diff
@@ -2,50 +2,50 @@
     "TwoFactorAuth": {
         "TwoFactorAuthentication": "Two-factor authentication",
         "TwoFAShort": "2FA",
-        "TwoFactorAuthenticationIntro": "%1$sTwo-factor authentication%2$s increase your account security by adding an additional layer of verification when you log in. Each time you log in you will not only be asked to provide your login and password, but also an additional authentication token which changes periodically and is generated for example on your mobile device. This means that even when someone knows your username and password, they still won't be able to log in unless they have access to your mobile device for example.",
-        "TwoFactorAuthenticationIsEnabled": "Two-factor authentication is currently enabled.",
-        "TwoFactorAuthenticationIsDisabled": "Two-factor authentication is currently disabled.",
-        "TwoFactorAuthenticationRequired": "Two-factor authentication is required to be enabled for everyone, you cannot disable it.",
-        "ConfigureDifferentDevice": "Configure a different device",
+        "TwoFactorAuthenticationIntro": "%1$sTwo-factor authentication%2$s increases account security by asking you for something generated on for example a mobile device (in addition to your password) to log in.",
```

Comment:
`something` sounds a bit unspecific and maybe not very helpful for someone who doesn't know what 2FA is. 
Matomo currently only supports 6 digit codes for 2FA generated by OTA app. In addition the user has a list with generated recovery codes that work once each.
Also I would maybe keep the other details it contained before like `This will prevent strangers from logging into your account even if they know your login and password.`

Reply:
I think if it is just right, (and I appreciate that it wasn't), it will be more clear.